src/Http/Wolverine.Http/Resources/WolverineProducesResponseTypeMetadata.cs

```diff
@@ -1,11 +1,8 @@
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
 using Microsoft.AspNetCore.Http.Metadata;
 
 namespace Wolverine.Http.Resources;
 
-internal class WolverineProducesResponseTypeMetadata : IProducesResponseTypeMetadata
+public class WolverineProducesResponseTypeMetadata : IProducesResponseTypeMetadata
```

Comment:
**COMMENT:** Without that one wouldn't be able to create its own `IHttpAware` implementation.